deploy/10-deployment.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: aws-cost-exporter
+  labels:
+    app: aws-cost-exporter
+    product: platform
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: aws-cost-exporter
+      product: platform
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        iam.amazonaws.com/role: arn:aws:iam::444914307613:role/prod-platform-aws-cost-exporter
+      labels:
+        app: aws-cost-exporter
+        product: platform
+    spec:
+      containers:
+      - env:
+         - name: TAG_PROJECT
+           valueFrom:
+             configMapKeyRef:
+               name: aws-cost-exporter
+               key: TAG_PROJECT
+         - name: PORT
+           valueFrom:
+             configMapKeyRef:
+               name: aws-cost-exporter
+               key: PORT
+        name: aws-cost-exporter
+        image: vivareal/aws-cost-exporter:v1.0.0
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 9150
+          protocol: TCP
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 9150
+            scheme: HTTP
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 30
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 9150
+            scheme: HTTP
+          initialDelaySeconds: 5
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 10
+        resources:
+            requests:
+              memory: 150Mi
+              cpu: 100m
+            limits:
+              memory: 150Mi
+              cpu: 100m
```

Comment:
requests should be lower or limits should be higher, it's not good practice to keep the same value for both